### ISSUE TITLE
Validation loss is too high ..Am I calculating the loss properly?

### ISSUE THREAD

**Issue by singhabhinav** [TorchEnsemble-Community/Ensemble-Pytorch#147]
Hi,

I am using VotingClassifier and logging out training and validation loss in TensorBoard to find out if my model is overfitting or underfitting. I had to modify the VotingClassifier code to calculate the validation loss and store in the tensorboard logs. Here is my logic which I am using for calculating validation loss.

```
     if test_loader: 
           validation_loss = self.evaluate(test_loader, return_loss = True):
           # log this validation_loss to TensorBoard
```

Below are my TensorBoard charts..We can see that validation loss is too high even though validation accuracy is very good. I am using cross entropy as criteria and the number of estimators are 10. Can you please let me know if my logic of calculating validation loss is right in VotingClassifier

![Screenshot 2023-03-05 at 8 40 00 PM](https://user-images.githubusercontent.com/10808722/224422738-6fc9851d-8d51-4876-b1fc-df22b03c12dd.png)
